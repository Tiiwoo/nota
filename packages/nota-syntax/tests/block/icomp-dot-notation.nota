.@span{lorem}
ipsum
dolor @span{sit}