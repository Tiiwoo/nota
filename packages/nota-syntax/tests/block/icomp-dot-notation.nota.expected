el("p", {}, el("span", {}, "lorem"), "\nipsum\ndolor ", el("span", {}, "sit"));